lib/ansible/plugins/inventory/vultr.py

```diff
@@ -0,0 +1,151 @@
+# (c) 2018, Yanis Guenane <yanis+ansible@guenane.org>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+    name: vultr
+    plugin_type: inventory
+    author:
+      - "Yanis Guenane (@Spredzy)"
+    short_description: Vultr inventory source
+    description:
+        - Get inventory hosts from Vultr public cloud.
+        - Uses C(api_config), C(~/.vultr.ini), C(./vultr.ini) or VULTR_API_CONFIG path to config file.
+    options:
+        plugin:
+            description: token that ensures this is a source file for the 'vultr' plugin.
+            required: True
+            choices: ['vultr']
+        api_account:
+            description: Specify the account to be used.
+            default: default
+        api_config:
+            description: Path to the vultr configuration file. If not specified will be taken from regular Vultr configuration.
+            env:
+              - name: VULTR_API_CONFIG
+        api_key:
+            description: Vultr API key. If not specified will be taken from regular Vultr configuration.
+            env:
+              - name: VULTR_API_KEY
+        hostname:
+            description: Field to match the hostname
+            type: string
+            default: v4_main_ip
+            choices:
+                - v4_main_ip
+                - v6_main_ip
+                - name
```

Comment:
`name` refers to the [`label` field](https://www.vultr.com/api/#server_server_list) returned by the Vultr API, that should be documented.